### ISSUE TITLE
v1.12.0 error running codegen on Postgres

### ISSUE THREAD

**Issue by andrewhughson** [rmp135/sql-ts#93]
After upgrading to 1.12.0, running `sql-ts` on `node16` throws an error when targeting a Postgres 12 database:

```
/fake_project/node_modules/tarn/dist/PendingOperation.js:25
        this.reject(new Error('aborted'));
                    ^

Error: aborted
    at PendingOperation.abort (/fake_project/node_modules/tarn/dist/PendingOperation.js:25:21)
    at /fake_project/node_modules/tarn/dist/Pool.js:208:25
    at Array.map (<anonymous>)
    at /fake_project/node_modules/tarn/dist/Pool.js:207:53
    at async Client_PG.destroy (/fake_project/node_modules/knex/lib/client.js:331:9)
```

I haven't tried other databases or other versions of Postgres, so I'm not sure if it's a version incompatibility or a bug in `sql-ts`.

1.11.0 works fine, keeping all other variables the same.

**Comment by andrewhughson** [comment]
Could possibly be related to this: https://github.com/rmp135/sql-ts/commit/26d641074f66b5998b7d9f3d68f04850d9a67d82#diff-e03a8eab1da029d3307ec758e43914c5b411491a425a86f30036c9069deec27dR20

**Comment by rmp135** [comment]
It is. It's not resolving the function before destroying the connection. Will have a fix out shortly.

**Comment by rmp135** [comment]
A fix has been released in v1.12.1.
